packages/back-end/src/controllers/reports.ts

```diff
@@ -189,15 +189,17 @@ export async function refreshReport(
   req: AuthRequest<null, { id: string }, { force?: string }>,
   res: Response
 ) {
-  req.checkPermissions("runQueries", "");
-
   const { org } = getOrgFromReq(req);
   const report = await getReportById(org.id, req.params.id);
 
   if (!report) {
     throw new Error("Unknown report id");
   }
 
+  const experiment = await getExperimentById(org.id, report.experimentId || "");
```

Comment:
Only do this if `report.experimentId` exists, since it's not always set.  Otherwise, it will do an unnecessary Mongo query